util/chplenv/utils.py

```diff
@@ -62,3 +62,20 @@ def run_command(command, stdout=True, stderr=False, cmd_input=None):
         else:
             return ''
 
+def run_live_command(command):
+    """Run a command, writing its output live instead of after the process is
```

Comment:
**comment nit-pick:**

 It's really *yielding* the merged output (stdout/stderr) as the process runs, rather than *writing* it live. 

**idea:**

A potential alternative to the iterator approach: Write the merged output to stdout within the function and return the output once the process completes, providing a simple interface similar to `run_command`. 

You could take it a step further and make this function accessible through a `live=False` argument in `run_command` and avoid the hard decision of choosing a name altogether.